website/pages/textarea.mdx

```diff
@@ -41,6 +41,6 @@ The Textarea composes the [Input](/input) component
 
 ## Event
 
-| Name   | Description                                                   |
-| ------ | ------------------------------------------------------------- |
-| change | Handles what happens when the content of the textarea changes |
+| Name   | Description                                                    |
+| ------ | -------------------------------------------------------------- |
+| change | Handles what happens when the content of the text area changes |
```

Comment:
This should also be left as is because it's derived from the native `<textarea />` tag
```suggestion
| change | Handles what happens when the content of the textarea changes |
```